src/adk/schema/applications/network.json

```diff
@@ -1,22 +1,22 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "properties": {
     "networks": {
       "type": "array",
-      "items": [
-        {
-          "type": "string"
-        }
-      ]
+      "items": {
+        "type": "string",
+        "readOnly": true,
+        "pattern": "^[-a-zA-Z0-9_]+$"
```

Comment:
Why does the regex for these names enforce a beginning and end of line (^and $) when this is not enforced for roles?

Reply:
^ matches begin of string
$ matches end of string
I copied the pattern because it is defined in the schema of the api-router.
the network names added here are slugs of network names we define (they are lower case